kahypar/io/hypergraph_io.h

```diff
@@ -391,7 +472,7 @@ static inline void readPartitionFile(const std::string& filename, std::vector<Pa
     }
     file.close();
   } else {
-    std::cerr << "Error: File not found: " << std::endl;
+    ERROR("File not found.");
```

Comment:
Would it make sense to use a `CheckedIStream` here as well?

Reply:
(as above)